assets/javascript/game.js

```diff
@@ -47,30 +54,61 @@ function givePayment(value) {
   updateMoney();
 }
 
-function improveSelf() {
-  if(money >= (Math.pow(click_value, 2))) {
-    money = money - (Math.pow(click_value, 2));
+function improveDeveloper(event) {
+  let developer_id = event.target.parentNode.parentNode.id;
+  let developer = developers.find(developer => developer.id === developer_id);
+  let price = developer.price();
+  if(money >= price) {
+    money = money - price;
     updateMoney();
-    click_value++;
+    developer.levelUp(1);
+    updateDeveloper(developer);
+    if(developer.level === 1) {
+      employees.innerHTML += '<img alt="' + developer.name + '" class="developer" src="assets/images/employees/' + developer.id + '.png">';
+    }
   }
 }
 
-function hireDeveloper() {
-  let developer = developers[helper_count];
-  if(money >= developer.price) {
-    money = money - developer.price;
-    updateMoney();
-    helper_count ++;
-    addNewDeveloper(developer);
+function updateDeveloper(developer) {
+  let developerElement = document.getElementById(developer.id);
+  let developerPosition = developers.indexOf(developer);
+  let developerAttributes = developerElement.childNodes[3];
+  developerElement.childNodes[1].classList = 'developer__icon';
+  developerAttributes.childNodes[1].classList = 'developer__attribute';
+  developerAttributes.childNodes[3].innerHTML = 'Level: ' + developer.level;
+  developerAttributes.childNodes[3].classList = 'developer__attribute';
+  developerAttributes.childNodes[5].innerHTML = 'Lines/Click: ' + developer.performance();
+  developerAttributes.childNodes[5].classList = 'developer__attribute';
+  developerAttributes.childNodes[7].innerHTML = 'Price: ' + developer.price() + '$';
+  developerAttributes.childNodes[7].classList = 'developer__attribute';
+  developerAttributes.childNodes[9].innerHTML = 'Improve skills';
+  if(developersElement.lastElementChild === developerElement && developerPosition < developers.length-1) {
+    addNewMenuDeveloper(developers[developerPosition+1]);
   }
 }
 
 function click() {
-  writtenLines = writtenLines + click_value;
+  writtenLines = writtenLines + mainDeveloper.performance();
   updatePerformance();
 }
 
-function addNewDeveloper(developer) {
-  developer.level = 1;
-  employers.innerHTML += '<img alt="' + developer.name + '" class="character" src="assets/images/employers/' + developer.imageName + '.png">'
+function addNewMenuDeveloper(developer) {
+  developersElement.insertAdjacentHTML( "beforeend", "<div id="+ developer.id +" class=\"developer__menu\">\n" +
```

Comment:
Why are u using `insertAdjacentHTML` 

Pls consider using https://www.w3schools.com/jsref/met_node_appendchild.asp

Reply:
And on HTML please consider too use again template strings